x-pack/solutions/observability/plugins/streams/server/routes/streams/crud/route.ts

```diff
@@ -145,6 +147,14 @@ export const editStreamRoute = createServerRoute({
   handler: async ({ params, request, getScopedClients }): Promise<UpsertStreamResponse> => {
     const { streamsClient } = await getScopedClients({ request });
 
+    if (!(await streamsClient.isStreamsEnabled())) {
+      throw badData('Streams are not enabled');
```

Comment:
badData results in a HTTP 422, since the request is syntactically correct and would pass our validation but there is a semantic error of streams not being enabled yet.